metals/src/main/scala/scala/meta/internal/metals/Embedded.scala

```diff
@@ -30,6 +33,23 @@ final class Embedded(
 
   override def cancel(): Unit = {
     presentationCompilers.clear()
+    mdocs.clear()
+  }
+
+  private val mdocs: TrieMap[String, URLClassLoader] =
```

Comment:
Can we move this value up to the top of the file?

Reply:
Done